Framework/HistogramData/test/ScalableTest.h

```diff
@@ -60,6 +78,14 @@ class ScalableTest : public CxxTest::TestSuite {
     TS_ASSERT_DELTA(lhs[1], 0.6, 1e-14);
   }
 
+  void test_divide_reverse_order() {
```

Comment:
Right, it makes sense in some cases, but not in others. It would be correct for, e.g., `Frequencies`, but wrong for `Counts` (or rather the output would not be `Counts` in that case). Should we remove it?